src/components/activityEventElems/AnyProjectEvent.tsx

```diff
@@ -0,0 +1,88 @@
+import { ProjectEventsQuery } from 'generated/graphql'
+
+import AddToBalanceEventElem from './AddToBalanceEventElem'
+import BurnEventElem from './BurnEventElem'
+import DeployedERC20EventElem from './DeployedERC20EventElem'
+import PayEventElem from './PayEventElem'
+import ProjectCreateEventElem from './ProjectCreateEventElem'
+import RedeemEventElem from './RedeemEventElem'
+import ReservesEventElem from './v1/ReservesEventElem'
+import TapEventElem from './v1/TapEventElem'
+import V1ConfigureEventElem from './v1/V1ConfigureEventElem'
+import ConfigureEventElem from './v2v3/ConfigureEventElem'
+import DeployETHERC20ProjectPayerEventElem from './v2v3/DeployETHERC20ProjectPayerEventElem'
+import DistributePayoutsElem from './v2v3/DistributePayoutsElem'
+import DistributeReservedTokensEventElem from './v2v3/DistributeReservedTokensElem'
+import SetFundAccessConstraintsEventElem from './v2v3/SetFundAccessConstraintsEventElem'
+
+/**
+ * Returns a formatted element for any project event
+ * @param event Project event object
+ * @param tokenSymbol Must be defined for projectEvents with a `burnEvent`
+ */
+export function AnyProjectEvent({
+  event,
+  tokenSymbol,
+}: {
+  event: ProjectEventsQuery['projectEvents'][number]
+  tokenSymbol: string | undefined
+}) {
+  if (event.addToBalanceEvent) {
```

Comment:
this is consolidation of logic that was largely duplicated across v1 and v2v3 dashboard pages. not directly necessary for the apollo integration, but felt like the right time to do this and "complete" all project activity-related cleanup on both dashboards